prod/westeurope/support/functions_backoffice/function_app/terragrunt.hcl

```diff
@@ -92,6 +91,11 @@ inputs = {
     }
   }
 
+  # TODO: use a dependency to resolve the origin
+  cors = {
+    allowed_origins = ["https://backoffice.io.italia.it"]
+  }
```

Comment:
lgtm, once CORS is configured on a test function I've noticed it replies back Access-Control-Allow-Header with whatever Access-Control-Request-Headers is sent from the client.
I expect the JS to set the Authorization header explicitly with "xhr.setRequestHeader" so that the flow would work also on preflighted request.

Reply:
thanks @pp-ps ... actually I've already applied this small change. We can test it straight in prod once the code of the function will be released.